doc/news/changes.h

```diff
@@ -197,6 +197,15 @@ inconvenience this causes.
 
 <ol>
 
+ <li> Improved: The parallel loops over the deal.II vector class have been
```

Comment:
maybe write this as "The parallel loops used in the deal.II Vector class for vector-vector operations have been..."